### DISCUSSION TITLE
Openthread Quality Dashboard

### DISCUSSION THREAD

**Discussion by farhangj** [openthread/openthread#10226] [Q&A]
Hi,
Wondering if there are newer versions of this report from 2019 that I can access.

https://openthread.io/testing/OpenThread_Quality_Dashboard.pdf

Quite useful to know the quality of the platform.

Thanks

**Comment by jwhui** [comment]
This was previously answered here: https://github.com/orgs/openthread/discussions/9662#discussioncomment-7710065
